src/Handler/Git.hs

```diff
@@ -234,8 +239,47 @@ gitPrintHandler (NHGit (GitRepoTrackable dir' _locSpec) nhg) = do
   liftIO $ putStrLn $ T.unpack $ format
 
 processGitNonInteractive :: GitRepoTrackable -> R.Reddup  ()
-processGitNonInteractive grTrack =
-  (lift $ checkGitProblems grTrack) >>= gitPrintHandler
+processGitNonInteractive grTrack = do
+  lift $ checkGitProblems grTrack >>= gitPrintHandler
+
+processGitNonInteractiveForce :: GitRepoTrackable -> R.Reddup ()
+processGitNonInteractiveForce grTrack =
+    lift . forever $
+        checkGitProblems grTrack >>= tryFixGitProblem
+
+tryFixGitProblem :: NHGit -> Tu.Shell ()
+tryFixGitProblem nh@(NHGit (GitRepoTrackable _dir _locSpec) nhg) = do
+    case nhg of
+        NHStatus (GP.Added _) ->
+            addAndWipCommit
+        NHStatus (GP.AddedAndModified _) ->
+            addAndWipCommit
+        NHStatus (GP.Staged _) ->
+            addAndWipCommit
+        NHStatus (GP.Unstaged _) ->
+            addAndWipCommit
+        NHStatus (GP.StagedAndUnstaged _) ->
+            addAndWipCommit
+        NHStatus (GP.Untracked _) ->
+            addAndWipCommit
+        NHStatus (GP.Deleted _) ->
+            addAndWipCommit
+        NHUnpushedBranch branch -> do
+            targetAndMerge <- (,) <$> readPushTarget branch <*> readMerge branch
+            let (GP.GitBranch branchName) = branch
+            liftIO $ gitPushCmd branchName (go targetAndMerge) mempty
+            pure ()
+        _ -> do
+            gitPrintHandler nh
+            Tu.mzero
+  where
+    go :: (Maybe a, Maybe a) -> Maybe (a, a)
+    go (Just a, Just b) = Just (a, b)
+    go _ = Nothing
+
+addAndWipCommit :: Tu.MonadIO m => m ()
+addAndWipCommit =
+    Tu.sh $ Tu.inshell "git add .; git commit -m 'reddup auto WIP commit'" Tu.empty
```

Comment:
so this is really minor, but I do have a thing that already does a wip commit in Handler.Git line 96.